### PR TITLE
Resnet Minor Polished

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

File: chapter_convolutional-modern/resnet.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ b1 = tf.keras.models.Sequential([
     tf.keras.layers.MaxPool2D(pool_size=3, strides=2, padding='same')])
 ```
 
-GoogLeNet 在后面接了 4 个由 Inception 块组成的模块。
+GoogLeNet 在后面接了 4 个由初始块组成的模块。
```
Comment (astonzhang) [inline chapter_convolutional-modern/resnet.md:232]: 不要硬翻Inception块

**Comment by mli** [comment]
Job d2l-zh/PR-612/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-612/

**Comment by mli** [comment]
Job d2l-zh/PR-612/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-612/

**Comment by mli** [comment]
Job d2l-zh/PR-612/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-612/

**Comment by mli** [comment]
Job d2l-zh/PR-612/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-612/

**Pull request merged**